packages/bpk-docs/src/layouts/NeoSideNav/SideNavLayout.scss

```diff
@@ -45,6 +54,10 @@
     background-color: $bpk-color-gray-900;
     color: $bpk-color-white;
 
+    @include bpk-breakpoint-tablet {
+      display: none;
```

Comment:
At the moment I am using css `display: none` to hide and show the mobile / desktop navigation components as needed. As discussed here https://github.com/Skyscanner/backpack/pull/644 there may be a better way to do this for screen readers etc